integration/test/dashboard_json_test.go

```diff
@@ -40,7 +40,15 @@ var zapiCounterMap = map[string]struct{}{
 
 // restCounterMap are additional counters, above and beyond the ones from counterMap, which should be excluded from Rest
 var restCounterMap = map[string]struct{}{
-	"aggr_snapshot_inode_used_percent": {},
+	"aggr_snapshot_inode_used_percent":                {},
+	"external_service_op_num_not_found_responses":     {},
```

Comment:
since these are included in `excludeCounters` they shouldn't be needed here too

Reply:
Done